library_generation/generate_showcase_integration_test.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source $SCRIPT_DIR/utilities.sh
+SDK_WORKSPACE=$SCRIPT_DIR/../WORKSPACE
+GAX_PROPERTIES=$SCRIPT_DIR/../gax-java/dependencies.properties
+SHOWCASE_BUILD_FILE=$SCRIPT_DIR/../showcase/BUILD.bazel
+
+cd $SCRIPT_DIR
+rm -rdf output
+
+# clone gapic-showcase
+if [ ! -d schema ]; then
+  git clone https://github.com/googleapis/gapic-showcase
+  pushd gapic-showcase
+  showcase_version=$(get_version_from_WORKSPACE "_showcase_version" $SDK_WORKSPACE)
+  git checkout "v$showcase_version"
+  mv schema ..
+  popd
+  rm -rdf gapic-showcase
+fi
+if [ ! -d google ];then
+  git clone https://github.com/googleapis/googleapis
+  mv googleapis/google .
+  mv googleapis/WORKSPACE ..
+  rm -rdf googleapis
+fi
+
+GOOGLEAPIS_WORKSPACE=$SCRIPT_DIR/WORKSPACE
+
+ggj_version=$(get_version_from_WORKSPACE _gapic_generator_java_version $SDK_WORKSPACE)
+if [ $(echo $ggj_version | grep 'SNAPSHOT' | wc -l) -gt 0 ]; then
+  echo 'This repo is at a snapshot version. Installing locally...'
+  pushd $SCRIPT_DIR/..
+  mvn clean install -DskipTests -Dclirr.skip
```

Comment:
I don't think we want to run `mvn` command in this shell script, we can rely on the behavior of `generate_library.sh` to handle it, just error out if it does not exist.

Reply:
Removed